framework/graphics/fps_info.cpp

```diff
@@ -26,6 +26,7 @@
 #include "util/date_time.h"
 #include "util/logging.h"
 
+#define __STDC_FORMAT_MACROS
```

Comment:
We use the format macros everywhere with no errors without this #define.  Is this specific to a compiler or system headers version?  It appears from this one web page https://www.gnu.org/software/gnulib/manual/html_node/inttypes_002eh.html that you'd only need this on C++ compilers before C++11, and we specify C++14 support in BUILD.md.
Can you tell me more about why you had to add this?

Reply:
This is actually the only place in the gfxreconstruct code where inttypes.h is included. Elsewhere #include <cinttypes> is used, which seems to work fine without adding the extra define line above. So I'll change it to use that instead.